contracts/test/StakeTable.t.sol

```diff
@@ -110,9 +133,9 @@ contract StakeTable_Test is Test {
             stakeTableAmountComm: 0,
             threshold: 0
         });
-        lightClientContract = new LightClientTest(genesis,10);
+        lightClientContract = new LightClientTest(genesis, 10);
         address lightClientAddress = address(lightClientContract);
-        stakeTable = new S(address(token),lightClientAddress);
+        stakeTable = new S(address(token), lightClientAddress, 10);
```

Comment:
let's remove L137, no point declaring a new variable here.

Reply:
Yes, done in 06b93b3 .